image/scanner/dump/genesis_manifests.json

```diff
@@ -5,418 +5,6 @@
       "timestamp": "2019-11-20T16:43:21.335508-08:00",
       "baseForOfflineDumps": true
     },
```

Comment:
I'm guessing this is the base so we cannot remove it, just want to confirm.

Reply:
Yea it used in the offline dump, we could (perhaps should) move it - but not knowing the full implications thought this iteration we should keep it.